wpeverest/everest-forms
# Feature - License settings page

### All Submissions:






### Changes proposed in this Pull Request:

Previously, there was no option to view and manipulate the licence key. This PR adds the feature to Activate/Deactivate and view the license plan along with its expiry date.

Closes # .

### How to test the changes in this Pull Request:

1. Go to **Everest Forms Settings > License**
2. You can view the field to enter the **License Key**
3. After that you can activate the license.
4. If your license is already activated, then you can deactivate it and check the license plan along with its expiry date.

[PR Dependency](https://github.com/wpeverest/everest-forms-pro/pull/959)

### Types of changes:




### Other information:




### Changelog entry

> Feature - License settings page.
